cvat/apps/dataset_manager/tests/assets/annotations.json

```diff
@@ -574,6 +574,24 @@
       }
     ]
   },
+  "MOT 1.1 shapes only": {
+    "version": 0,
+    "tags": [],
+    "shapes": [
+      {
+        "type": "rectangle",
+        "occluded": false,
+        "z_order": 0,
+        "points": [9.4, 12.09, 17.2, 18.19],
+        "frame": 0,
+        "label_id": null,
+        "group": 0,
+        "source": "manual",
+        "attributes": []
+      }
```

Comment:
I don't think this tests for the bug that this PR is meant to be fixing - `occluded` is false, and there's no `visibility` attribute. There should be a variety of annotations with different values of these attributes.

Reply:
Attributes are added when creating annotations (in the `_create_annotations` function)

Reply:
OK, but what about `occluded`? That's not an attribute.

Reply:
What's wrong with the `occluded` attribute? There is a problem with `visibility` and `ignored`.

Reply:
Oh, okay. I got confused by the other changes.